src/index.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import Typography from "@material-ui/core/Typography";
+import TodoForm from "./TodoForm";
+import TodoList from "./TodoList";
+import useTodoState from "./useTodoState";
+import "./styles.css";
+import logo from "./Logo.jpeg";
+import Footer from "./Footer";
+
+const App = () => {
+  const { todos, addTodo, deleteTodo } = useTodoState([]);
+
+  return (
+    <div className="App">
+      <img alt="Logo" src={logo} />
+      <Typography component="h1" variant="h2">
+        Nicole To Do List
+      </Typography>
+
+      <TodoForm
+        saveTodo={(todoText) => {
+          const trimmedText = todoText.trim();
+
+          if (trimmedText.length > 0) {
+            addTodo(trimmedText);
+          }
+        }}
```

Comment:
The component itself could do this check already. So this listener wouldn't have to check this here.

Reply:
This is still pending